integreat_cms/cms/templates/pages/_page_order_table.html

```diff
@@ -77,7 +78,7 @@
                     <tr class="border-2 border-blue-500 hover:bg-gray-100">
                         <td class="single_icon min">
                             <span class="drag text-gray-800 block py-3 px-2 cursor-move"
-                                  draggable="true">
+                                  draggable="{% if can_edit_page %}true{% else %}false{% endif %}">
```

Comment:
I think it could maybe even make sense to remove the "move-cursor" and maybe even remove the move-icon.

Reply:
Applied!